code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -275,3 +275,11 @@
 	icon_state = "pill8"
 	list_reagents = list(/datum/reagent/iron = 30)
 	rename_with_volume = TRUE
+
+/obj/item/reagent_containers/pill/methos
+	name = "Methos(TM)"
+	desc = "A sugary threat."
+	icon_state = "methos"
+	list_reagents = list(/datum/reagent/drug/methamphetamine = 1, /datum/reagent/consumable/mentonol = 10)

```

Comment:
Why are you like this?

Reply:
whats wrong?

Reply:
Why don't you put random packs of bath salts in maint while you are at it.
I want to know what's right about this first. Meth is *essentially* a lategame chem which isn't that easily aquired without chemical equipment or botany.
You can do a lot of shitty stuff with even small quantities of meth and the only other way of aquiring it this easily is with addict quirk, which has obvious drawbacks and is RNG dependent.
The pun is also really bad.
I wouldn't mind if this was something you can use as pseudo to cook up ghetto meth but just plain meth invalidates too many other tasks.

Reply:
is 0.4 meth ok? (1 tick) and meth isnt a lategame chem, just look at shit like rdx, tatp, life and then look at meth made without ever swapping a beaker and an igniter

Reply:
why are you like this

Reply:
i ll go for "I wouldn't mind if this was something you can use as pseudo to cook up ghetto meth " suggestion